apps/weshipit/components/layout.tsx

```diff
@@ -14,19 +15,24 @@ export function Layout({
   withHeader = false,
   withFooter = false,
   withContainer = false,
+  noBackground = false,
   withAccessoryRight,
   seoTitle,
   seoDescription,
   ogImageTitle,
 }: LayoutProps) {
+  let backgroundColor = 'bg-gray-100 dark:bg-black';
```

Comment:
it's working as a first iteration but it's a bit hacky. 

If you want to add style use `cva` with a variant. That way we will have more control over time.